geojson/geojson_s2_util.go

```diff
@@ -40,6 +40,20 @@ func polylineIntersectsPoint(pls []*s2.Polyline,
 
 func polylineIntersectsPolygons(pls []*s2.Polyline,
 	s2pgns []*s2.Polygon) bool {
+	// Early exit path for lines completely contained within the polygon.
```

Comment:
comment is incomplete? 
lines partially contained also get exited here.

"early exit if the polygon contains either of the line's vertices."